src/pages/guides/uxp_guide/uxp-misc/manifest-v5/index.md

```diff
@@ -0,0 +1,282 @@
+
+## Changes to entry points
+The changes to `entryPoints` add flexibility in specifying the initial view/location of plugin panels.
+
+Key | Type | Description | Change
+----|------|------------ |------
+`id` | string | Unique identifier for the entry point. This `id` will also be mapped to entrypoints defined in your plugin code. | Passed as the id for the uxpcommand, uxpshowview (New V5) and  uxphideview (New V5) events.
+`hostUIContext` | object | Use [this property](#hostuicontext) to customize how entrypoints appear. | New in v5.
+`featureFlags` | object | Indicates feature flags in use. | `enableWebview` boolean flag. 
+
+### HostUIContext 
+By default, panels from the same plugin are grouped together by the host app. If more control is needed, developers can use this property.
+
+Key | Type | Default | Description
+----|------|------------|---
+`hideFromMenu` | boolean | `false` | If `true`, the entrypoint is hidden from the plugins menu.
+`hideFromPluginsPanel` | boolean | `false` | If `true`, the entrypoint is removed from the plugins panel (launchpad).
+`initialLocation` | string | plugin ID | Plugins are grouped together upon ID by default. You can specify a new ID to group subsets of panels together. 
+`triggerEvents` | string[] | -- | Specifies if the entrypoint should be triggered by host-determined events. 
+
+## Plugin Permissions
+Plugins using Manifest v5 will enjoy the enhancements in security with the introduction of new permissions model. Users will be asked for their consent when your plugin attempts to use openExternal, openPath, and sp-link/anchor tags. For everything else, consent is assumed given at install time.
+
+Starting with v5, any permissions not explicitly declared in the manifest are by default not granted. 
+
+### Network
+You have to specify unlock the `network` module in `requiredPermissions` by defining domains from which network requests are allowed. `domains` accepts an array of strings or a singular string `"all"`.
+```json
+{
+    "requiredPermissions": {
+        "network": {
+            "domains": [
+                "https://source.unsplash.com",         
+            ]
+        }
+    }
+}
+```
+```html
+<img src='https://source.unsplash.com/random'>
+```
+### Clipboard
+To grant read and/or write access to the system clipboard, declare it in `requiredPermissions`.
+`clipboard` accepts:
+* `readAndWrite` for read/write access
+* `read` for read-only access. 
+
+```json
+{
+    "requiredPermissions": {
+        "clipboard": "readAndWrite"
+    }
+}
+```
+ 
+```js
+const clipboard = navigator.clipboard;
+const dataTransferProviders = {
+    'text/plain': 'Sample text'
+};
+clipboard.writeText(dataTransferProviders).then(
+    (result) => {...},
+    (error) => {...}
+);
+```
+
+### Local Filesystem
+`localFileSystem` accepts:
+* `request`: Allows the plugin to access the local file system using token provided by UXP.
```

Comment:
Allows the plugin to access files on the local file system. For files outside of the plugin's data, temporary, and code folders, the user will be prompted for consent.

Reply:
`localFileSystem` accepts:
* `request`: Allows the plugin to access files on the local file system. For files outside of the plugin's data, temporary, and code folders, the user will be prompted for consent.
* `plugin`: Allows the plugin to access the plugin's sandbox, but does not enable file picker APIs.

Reply:
localFileSystem also accepts 'fullAccess' that gives access to any file on the user's machine with the use of file scheme. Example: `<img id="im1" src="file:/Users/pakrishn/Downloads/CovidVaccineCard.png">`